src/test/scala/dpla/ingestion3/enrichments/StringUtilsTest.scala

```diff
@@ -198,4 +198,35 @@ class StringUtilsTest extends FlatSpec with BeforeAndAfter {
     val enrichedValue = originalValue.findAndRemoveAll(stopWords)
     assert(enrichedValue === "photograph")
   }
+
```

Comment:
@guegueng Can you look at these tests and let me know if you see any obvious cases not addressed?